model/http/registry.yaml

```
@@ -27,7 +27,15 @@ groups:
           The attribute value MUST consist of either multiple header values as an array of strings
           or a single-item array containing a possibly comma-concatenated string, depending on the way
           the HTTP library provides access to headers.
-        examples: ['http.request.header.content-type=["application/json"]', 'http.request.header.x-forwarded-for=["1.2.3.4", "1.2.3.5"]']
+
+          Examples:
+
+          - A header `Content-Type: application/json` should be recorded as the `http.request.header.content-type`
```

(and other places if you agree)
```suggestion
          - A header `Content-Type: application/json` SHOULD be recorded as the `http.request.header.content-type`
```